roles/reth/defaults/main.yaml

```diff
@@ -61,3 +61,69 @@ reth_container_command_extra_args: []
 
 # Default image pull policy
 reth_container_pull: false
+
+
+# rbuilder - https://github.com/flashbots/rbuilder
+reth_rbuilder_enabled: false
+reth_rbuilder_config_path: /data/rbuilder.toml
+reth_rbuilder_container_volumes:
+  - "{{ reth_rbuilder_config_path }}://rbuilder-config.toml:ro"
+reth_rbuilder_container_args:
+  - --rbuilder.config=/rbuilder-config.toml
+
+reth_rbuilder_config: |
+  log_json = true
+  log_level = "info,rbuilder=debug"
+  redacted_telemetry_server_port = 6061
+  redacted_telemetry_server_ip = "0.0.0.0"
+  full_telemetry_server_port = 6060
+  full_telemetry_server_ip = "0.0.0.0"
+
+  chain = "mainnet"
```

Comment:
is this line needed tho? mainnet sounds off or is this a preset that doesnt matter

Reply:
It's just some default preset, based on a config example from rbuilder: https://github.com/flashbots/rbuilder/blob/develop/config-live-example.toml 

The config has to anyway to be overwritten for any specific deployment case. It's just an example config.